webapp/src/components/meeting_settings/meeting_settings.jsx

```diff
@@ -17,8 +17,9 @@ export default class MeetingSettingsModal extends React.PureComponent {
         super(props);
 
         this.state = {
-            hashtag: '{{Jan02}}',
+            hashtagPrefix: 'Prefix',
             weekdays: [1],
+            dateFormat: '1-2',
```

Comment:
What does `1-2` mean?

Reply:
Great catch, seems like I missed this while separating commits, will fix this.